assets/stylesheets/shared/_color-schemes.scss

```diff
@@ -216,25 +216,43 @@
 	--color-link-900-rgb: #{hex-to-rgb( $muriel-blue-900 )};
 
 	--color-wpcom: #{$muriel-blue-500};
+	--color-wpcom-rgb: #{hex-to-rgb( $muriel-blue-500 )};
 	--color-jetpack: #{$green-jetpack};
-	--color-facebook: #{$color-facebook};
-	--color-twitter: #{$color-twitter};
-	--color-gplus: #{$color-gplus};
-	--color-tumblr: #{$color-tumblr};
-	--color-linkedin: #{$color-linkedin};
-	--color-path: #{$color-path};
-	--color-instagram: #{$color-instagram};
-	--color-eventbrite: #{$color-eventbrite};
-	--color-stumbleupon: #{$color-stumbleupon};
-	--color-reddit: #{$color-reddit};
-	--color-pinterest: #{$color-pinterest};
-	--color-pocket: #{$color-pocket};
-	--color-email: #{$color-email};
-	--color-print: #{$color-print};
-	--color-skype: #{$color-skype};
-	--color-telegram: #{$color-telegram};
-	--color-whatsapp: #{$color-whatsapp};
-	--color-wordpress-org: #{$color-wordpressOrg};
+	--color-email: #f8f8f8;
+	--color-email-rgb: #{hex-to-rgb( #f8f8f8 )};	
```

Comment:
We don't need any of these rgb variables

Reply:
Removed. :+1: